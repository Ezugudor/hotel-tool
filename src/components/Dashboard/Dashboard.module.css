.Header{
    background:#fff;
    padding:30px;
    font-size:30px
}

.StatsCont{
    display:flex;
    justify-content:space-between;
    padding:25px
}

.PlaceholderWrapper{
   display:flex;
   justify-content:space-around;
   padding-top:50px;
   /* height:80% */
}
.PlaceholderCont{
    align-self:center;
    width:70%;
    height:40vh;
    display:flex;
    flex-direction:column;
    align-items:center
}
.PlaceholderIconCont{
    padding:20px;
}

.PlaceholderIconCont i{
   font-size:7vw;
   color:#777
}

.PlaceholderCont h4{
    font-size:3vw;
    color:#999
 }
 .PlaceholderInfo{
    font-size:2vw;
    color:#aaa;
    text-align:center
 }
 .TableCont{
     /* background:red; */
     /* height:100vh */
     padding:0 25px
 }
 .TableHeader{
     display:flex;
     justify-content:space-between;
     background:#fff;
     padding:30px;
     margin-bottom:5px;
     align-items:center
 }
 .TableHeader input{
    background:#f1f1f1;
    border:1px solid #ccc;
    padding:10px;
    border-radius:4px
}
 .TableCont table{
     width:100%;

 }

 .TableCont table thead th, .TableCont table tbody td{
   background:#fff;
   padding:10px
    
}
.Todo{
    color:#aaa
}