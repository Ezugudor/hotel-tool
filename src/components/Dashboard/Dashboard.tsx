import React from "react";


export const Dashboard: React.FunctionComponent = () =>
    <div>

    </div>;
