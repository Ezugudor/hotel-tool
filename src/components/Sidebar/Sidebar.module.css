.Container{
    background:#8b60bd;
    height:100%;
    padding:50px;
    
}
.LogoCont{
   padding-bottom:30px
}
.Header{
   margin-bottom:60px
}

.HeaderText{
    margin:0;
    font-size:20px;
    text-align:center;
    background: #956cc4;
    padding:20px;
    color:#fff;
    font-weight:normal
}
.RequestCont{
    margin-bottom:100px
}