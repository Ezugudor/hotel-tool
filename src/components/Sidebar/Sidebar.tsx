import React from "react";
import { AvailableRoomForm } from "./AvailableRoomForm";
import { CustomerRequestSummary } from "./CustomerRequestSummary";
import styles from "./Sidebar.module.css"



export const Sidebar: React.FunctionComponent = () => {

    return (
        <div className={styles.Container}>
            <div className={styles.LogoCont}>

            </div>
            <div className={styles.Header}>
                <h1 className={styles.HeaderText}>Incoming Requests Info:</h1>
            </div>
            <div className={styles.RequestCont}>
                <CustomerRequestSummary />
            </div>

            <div className={styles.Header}>
                <h1 className={styles.HeaderText}>Available Rooms:</h1>
            </div>
            <div className={styles.FormCont}>
                <AvailableRoomForm />
            </div>
        </div>
    )
};
