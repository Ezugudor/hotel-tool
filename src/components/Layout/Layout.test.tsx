import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Layout } from '.';
import { Sidebar } from '../Sidebar';
import { Dashboard } from '../Dashboard';


let LayoutCont: ShallowWrapper;
let sideBarCont: ShallowWrapper;

beforeAll(() => {
    LayoutCont = shallow(<Layout />);
    sideBarCont = LayoutCont.find(Sidebar);

});


describe("Layout Component Test", () => {
    it('should match with existing snapshot', () => {
        expect(LayoutCont.html()).toMatchSnapshot();
    })

    it("should render single Layout component", () => {
        expect(LayoutCont).toHaveLength(1)
    })

    it("should render Sidebar component", () => {
        expect(sideBarCont).toHaveLength(1);
    })

    it("should render Dashboard component", () => {
        const dashboardCont = LayoutCont.find(Dashboard);
        expect(dashboardCont).toHaveLength(1);
    })


})
