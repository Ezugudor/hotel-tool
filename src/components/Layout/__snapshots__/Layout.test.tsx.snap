// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component Test should match with existing snapshot 1`] = `"<div><div></div><div></div></div>"`;
